### PR TITLE
Reduce the attribute cache interval for custom attributes to get reflected faster

### PR DESCRIPTION
**Pull request by suresh-prakash**

### CODE REVIEW

**Review by skjindal93 — APPROVED** [review]

File: gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java [hunk 1]
```diff
@@ -21,10 +24,15 @@
...
-        .setAttribute(columnIdentifierConverter.convert(source.getAttribute()))
-        .setOperator(operatorConverter.convert(source.getOperator()))
-        .setValue(literalConstantConverter.convert(source.getValue()))
-        .build();
+    final Builder builder =
+        AttributeUpdateOperation.newBuilder()
+            .setAttribute(columnIdentifierConverter.convert(source.getAttribute()))
+            .setOperator(operatorConverter.convert(source.getOperator()));
+
+    if (source.getOperator() != OPERATOR_UNSET) {
```
Comment (aaron-steinfeld) [inline gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java:32]: If the operator is unset, why are we setting it on line 30?

Comment (aaron-steinfeld) [reply gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java:32]: But why do we need to convert and set the operator? Is unset a special value? Usually if we get an unset value, we don't want to blindly set it in the proto, we want to leave it unset to support presence detection.

Comment (suresh-prakash) [reply gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java:32]: I think, I understood why we are confused here. 
This is a deliberately created enum value (not the proto generated one)
image

Comment (aaron-steinfeld) [reply gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/converter/UpdateOperationConverter.java:32]: Ah - got it, yes I thought this was the unspecified case! thanks for following up.

**Pull request merged**